persistence-cassandra-4.0/pom.xml

```diff
@@ -4,11 +4,10 @@
   <parent>
     <groupId>io.stargate</groupId>
     <artifactId>stargate</artifactId>
-    <version>0.0.13-SNAPSHOT</version>
+    <version>0.0.14-SNAPSHOT</version>
   </parent>
   <groupId>io.stargate.db.cassandra</groupId>
   <artifactId>persistence-cassandra-4.0</artifactId>
-  <version>0.0.6-SNAPSHOT</version>
```

Comment:
Removing since it turns out that maven _really_ does not like for parent and child versions to be out of whack.